Makefile

```diff
@@ -18,180 +18,68 @@
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.1.0
+DOCKER_IMAGE_DEVELOPER_TOOLS := cft/developer-tools
+REGISTRY_URL := gcr.io/cloud-foundation-cicd
 
-# All is the first target in the file so it will get picked up when you just run 'make' on its own
-.PHONY: all
-all: check generate_docs
-
-# Run all available linters
-.PHONY: check
-check: check_shell check_python check_golang check_terraform check_base_files test_check_headers check_headers check_trailing_whitespace
-
-# The .PHONY directive tells make that this isn't a real target and so
-# the presence of a file named 'check_shell' won't cause this target to stop
-# working
-.PHONY: check_shell
-check_shell: ## Lint shell scripts
-	@source test/make.sh && check_shell
-
-.PHONY: check_python
-check_python: ## Lint Python source files
-	@source test/make.sh && check_python
-
-.PHONY: check_golang
-check_golang: ## Lint Go source files
-	@source test/make.sh && golang
-
-.PHONY: check_terraform
-check_terraform: ## Lint Terraform source files
-	@source test/make.sh && check_terraform
-
-.PHONY: check_docker
-check_docker: ## Lint Dockerfiles
-	@source test/make.sh && docker
-
-.PHONY: check_base_files
-check_base_files:
-	@source test/make.sh && basefiles
-
-.PHONY: check_trailing_whitespace
-check_trailing_whitespace:
-	@source test/make.sh && check_trailing_whitespace
-
-.PHONY: test_check_headers
-test_check_headers:
-	@echo "Testing the validity of the header check"
-	@python test/test_verify_boilerplate.py
-
-.PHONY: check_headers
-check_headers: ## Check that source files have appropriate boilerplate
-	@source test/make.sh && check_headers
-
-.PHONY: test_migrate
-test_migrate:
-	@echo "Testing migrate script"
-	@python test/helpers/test_migrate.py
-
-.PHONY: test_preconditions
-test_preconditions:
-	@echo "Testing preconditions script"
-	@python test/scripts/preconditions/test_preconditions.py
-
-# Unit tests
-.PHONY: test_unit ## Run unit tests
-test_unit: test_migrate test_preconditions
-
-# Integration tests
-.PHONY: test_integration
-test_integration: ## Run integration tests
-	test/ci_integration.sh
-
-.PHONY: generate_docs
-generate_docs: ## Update README documentation for Terraform variables and outputs
-	@source test/make.sh && generate_docs
-
-.PHONY: release-new-version
-release-new-version:
-	@source helpers/release-new-version.sh
-
-# Run docker
+# Enter docker container for local development
 .PHONY: docker_run
-docker_run: ## Launch a shell within the Docker test environment
+docker_run:
 	docker run --rm -it \
-		-e BILLING_ACCOUNT_ID  \
-		-e DOMAIN \
-		-e FOLDER_ID \
-		-e GROUP_NAME \
-		-e ADMIN_ACCOUNT_EMAIL \
-		-e ORG_ID \
-		-e PROJECT_ID \
 		-e SERVICE_ACCOUNT_JSON \
-		-e GOOGLE_APPLICATION_CREDENTIALS=${CREDENTIALS_PATH} \
-		-v $(CURDIR):/cft/workdir \
-		${DOCKER_REPO_BASE_KITCHEN_TERRAFORM} \
-		/bin/bash -c "source test/ci_integration.sh && setup_environment && exec /bin/bash"
+		-v $(CURDIR):/workspace \
```

Comment:
Can we quote `$(CURDIR)` here? i.e. `-v "$(CURDIR)":/workspace`. This is necessary to run the tests on my machine.

Reply:
Updated

Reply:
@Jberlinsky can you open an issue for this against the module template?

Reply:
Yep! https://github.com/terraform-google-modules/terraform-google-module-template/issues/33